config/purposes/deep_well_96_stamping.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Entry point for LTHR pipeline from Beckman picked plates
+# Presents the user with a page to select a workflow, which
+# builds the appropriate Sequencescape submission
+LDW-96 Stock:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: false
+  :input_plate: true
+  :creator_class: LabwareCreators::Uncreatable
+  :presenter_class: Presenters::SubmissionPlatePresenter
+  :submission_options:
+    Stamp from Deep-Well to Shallow-Well:
+      template_name: 'Limber-Htp - Deep Well Plate Stamping'
+      request_options:
+        fragment_size_required_from: '50'
+        fragment_size_required_to: '800'
```

Comment:
Is this needed? If is a config they asked ok, but I would think is not?

Reply:
This is a good point.  Andrew suggested that it may be necessary to provide some kind of value, since it's also used in Cardinal (I think).  But I will try removing it and see if everything still works, because it's true that this was only our suspicion at the time of creating this file.

Reply:
I tried removing `request_options` entirely but that presents a stack trace explaining that you can't iterate with `each` on a `nil` object when trying to load the choose_workflow partial for the plate page due to this code:

```
<%= f.fields_for('request_options') do |ro| %>
  <% submission.request_options.each do |k,v| %>
    <%= ro.hidden_field k, value: v %>
  <% end %>
<% end %>
```

So I set `request_options: {}` which got a bit further but later said that "Request options can't be blank".  I'm pretty sure this error is coming from Sequencescape and not Limber.  However I can't find where in Sequencescape this is actually generated.

The best I could do was to use:

```
request_options:
  null_key: ''
```

which Sequencescape seems to ignore even though there are no other values it is expecting to see.  This feels about as hacky as providing unneeded keys that it does recognise but let me know what you think.